lib/llm/src/kv_router/indexer.rs

```diff
@@ -1304,65 +1318,107 @@ mod tests {
     #[tokio::test]
     #[apply(indexer_template)]
     async fn test_frequency(num_shards: usize, kv_block_size: usize) {
+        const ONE_MILLIS: Duration = Duration::from_millis(1);
```

Comment:
Any significance for giving a const to 1ms, but not to the other usages, ex: 50ms, 10ms, etc. used as Duration::from_millis(50) below?